### PR TITLE
Github authentication tutorial

### PR DESCRIPTION
**Pull request by leplatrem**
In the series of tutorials... here comes Github auth!

### CODE REVIEW

File: docs/tutorials/authentication-github.rst [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+        def _get_credentials(self, request):
+            authorization = request.headers.get('Authorization', '')
+            print('Check Github')
+
+
+Don't be scared by those lines. It just implements the necessary methods to match the `IAuthenticationPolicy <http://docs.pylonsproject.org/projects/pyramid/en/latest/api/interfaces.html#pyramid.interfaces.IAuthenticationPolicy>`_ Pyramid interface.
+
+
+Add it to Python path
```
Comment (Natim) [inline docs/tutorials/authentication-github.rst]: Why not tell people to use a virtualenv instead? Would it be easier for them?

File: docs/tutorials/authentication-github.rst [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>> import kinto_github
+    >>>
+
+
+Enable in configuration
+-----------------------
+
+:ref:`As explained in the settings section <configuration-authentication>`, just
+enable a new policy pointing to your python module:
```
Comment (Natim) [inline docs/tutorials/authentication-github.rst]: nit: Python with a capital P.

File: docs/tutorials/authentication-github.rst (hunk 1 above)
Comment (leplatrem) [inline docs/tutorials/authentication-github.rst]: But it does not save you the step of adding to Python path ?

Comment (Natim) [inline docs/tutorials/authentication-github.rst]: You can use the ``add_path_to_virtualenv`` command au wait it just works with ``pew`` - ``pew add /path/to/folder`` nevermind.

File: docs/tutorials/authentication-github.rst [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+    class GithubAuthenticationPolicy(CallbackAuthenticationPolicy):
+        def __init__(self, realm='Realm'):
+            self.realm = realm
+
+        def unauthenticated_userid(self, request):
+            user_id = self._get_credentials(request)
+            return user_id
+
+        def forget(self, request):
+            return [('WWW-Authenticate', 'Bearer realm="%s"' % self.realm)]
```
Comment (Natim) [inline docs/tutorials/authentication-github.rst]: It should probably be ``Github+Bearer`` here.

File: docs/tutorials/authentication-github.rst [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
```
Comment (Natim) [inline docs/tutorials/authentication-github.rst]: Same here, ``GITHUB_METHOD`` should be used.

File: docs/tutorials/authentication-github.rst [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+
+.. code-block:: http
+
+    HTTP/1.1 401 Unauthorized
+    Access-Control-Expose-Headers: Retry-After, Content-Length, Alert, Backoff
+    Content-Length: 110
+    Content-Type: application/json; charset=UTF-8
+    Date: Tue, 26 Jan 2016 11:07:05 GMT
+    Server: waitress
+    Www-Authenticate: Bearer realm="Realm"
```
Comment (Natim) [inline docs/tutorials/authentication-github.rst]: You should have Github+Bearer here now :)

Comment (leplatrem) [inline docs/tutorials/authentication-github.rst]: Good catch!

**Comment by Natim** [comment]
LGTM thanks :+1:

**Comment by ptgamr** [comment]
@leplatrem if I want to implement it the right way, where should I put the file `kinto_github.py` ? (sorry I'm not so familiar with Python)

**Comment by leplatrem** [comment]
Having it in the «Python path» should be enough. Did you try leaving it in the current folder for example?

You can also use the `PYTHONPATH` environment variable for example:

- `$ export PYTHONPATH=/home/your/path:$PYTHONPATH`
- `$ kinto start`

Let us know! (please continue mentioning my nick so that I get the notification more easily)

**Pull request merged**